### DISCUSSION TITLE
"Commercial use"  license inquiry

### DISCUSSION THREAD

**Discussion by VictorDidier** [angelolab/ark-analysis#1049] [Announcements]
I would like to test pixie on a test data generated by a company, I would also run the analysis on its IT infrastructure. My question is... would I need a commercial license for this trial even though no profit is intended ? or could I run a test without infringing the current license?

**Comment by ngreenwald** [comment]
Nope, no commercial license for that, go ahead.

**Comment by VictorDidier** [comment]
Great, thank you for your reply.
